ostd/src/arch/x86/trap.rs

```diff
@@ -70,7 +70,7 @@ extern "sysv64" fn trap_handler(f: &mut TrapFrame) {
 
 /// Handles page fault from user space.
 fn handle_user_page_fault(f: &mut TrapFrame, page_fault_addr: u64) {
-    let current_task = current_task().unwrap();
+    let current_task = Task::current().unwrap();
```

Comment:
How do you like the idea of changing the return type of `Task::current` from `Option<Arc<Task>>` to `Arc<Task>`. I think this would benefit almost all users of `Task::current`. For the rare cases, when `Task::current` may fail, we can let the user call another `Task::try_current` method. Of course. This can be done in a separate PR.

Reply:
I have no inclinations for this argument. I do not think it is a bad idea to panic if called in the interrupt contexts too. We can do it in the near future.

Reply:
ok

Reply:
sorry, I had a typo replying this

> I do not think it is a bad idea to panic if called in the "bootstrap" contexts too.

not "interrupt" contexts